tmt/steps/provision/podman.py

```diff
@@ -24,8 +23,14 @@ def __init__(self, data, step):
         self.container_name = None
         self.container_id = None
 
+        # Environment variables compatible with commands used
+        self.podman_env = [f'-e {env}' for env in self.opt('environment')]
+        self.shell_env = ' '.join(self.opt('environment'))
+
     def podman(self, command, **kwargs):
```

Comment:
This is somewhat subjective, but I'd do `*command` here,
so that 

```
self.container_id = self.podman(['run'] + self.podman_env + ['--name', self.container_name, ...
```

becomes

```
self.container_id = self.podman('run', *self.podman_env, '--name', self.container_name, ...
```

But feel free to dismiss this suggestion.

Reply:
I will leave it if you do not mind, `podman` function currently takes only one parameter as our `run` function. I would need to redo also that.